examples/delete_disks.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Show how to delete many disks created by the 'add_disks.py' example quickly.
+"""
+
+import time
+from contextlib import closing
+
+from ovirtsdk4 import types
+
+from helpers import common
+from helpers.common import progress
+from helpers.units import humansize, GiB
+
+parser = common.ArgumentParser(description="Delete disks created by the 'add_disks.py' example")
+
+parser.add_argument(
+    "--sd-name",
+    required=True,
+    help="Name of the storage domain.")
+
+args = parser.parse_args()
+
+def get_storage_domain_disk_service_by_desc(sd_service, disk_desc):
+    disks_service = sd_service.disks_service()
+    # StorageDomainDisksService.list has no 'search' parameter and ignores
+    # query={'name': 'spam'} so we have to do the filtering ourselves
+    disk = next((disk for disk in disks_service.list() if disk.description == disk_desc), None)
+    if disk is None:
+        return None
+    else:
+        return disks_service.disk_service(disk.id)
+
+progress("Connecting...")
+connection = common.create_connection(args)
+with closing(connection):
+    disks_service = connection.system_service().disks_service()
+    storage_domains_service = connection.system_service().storage_domains_service()
+    sd = storage_domains_service.list(search='name={}'.format(args.sd_name))[0]
+    sd_service = storage_domains_service.storage_domain_service(sd.id)
+
+    waiting = set()
+    count = len(sd_service.disks_service().list())
+    desc = "Created by add_disks.py"
+
+    for i in range(count):
```

Comment:
Why iterated on all disks in the system?!

Reply:
iterate the disks in the storage domain - I've created a unique sd for the 5000 disks, it fits me, not for the general use case.

Reply:
Code changed :)